.github/ISSUE_TEMPLATE.md

```diff
@@ -1,22 +1,23 @@
-**Please DO NOT report security vulnerabilities here. Send them to security@mautic.com instead.**
+[//]: # (READ THIS BEFORE PUBLISHING YOUR ISSUE)
 
-What type of report is this:
+[//]: # (Please post questions, support requests and feature requests/ideas at https://forum.mautic.org.)
+[//]: # (Strictly follow the instruction hereunder, otherwise your issue will be close for lack of information.)
+[//]: # (DO NOT report security vulnerabilities here. Send them to security@mautic.com.)
+
+Detail your bug report:
 
 | Q  | A
 | ---| ---
-| Bug report? | 
-| Feature request? | 
-| Enhancement? | 
+| Confirm it is a Bug report |
+| Mautic version | 
+| PHP version | 
+
+[//]: # (Feature request? ---> Go to forum please ---> https://forum.mautic.org.)
```

Comment:
I would maybe direct them specifically to the features category https://forum.mautic.org/c/ideas